src/main/java/com/github/dockerjava/api/model/SwarmCAConfig.java

```diff
@@ -0,0 +1,63 @@
+package com.github.dockerjava.api.model;
```

Comment:
maybe create and move to `swarm` package? I guess model folder will be huge soon.